database/src/main/resources/db/migration/V102__oppdater_bosituasjon_periode.sql

```diff
@@ -0,0 +1,7 @@
+update grunnlag_bosituasjon g
+set fraOgMed = (stønadsperiode -> 'periode' ->> 'fraOgMed')::date,
+    tilogmed = (stønadsperiode -> 'periode' ->> 'tilOgMed')::date
```

Comment:
Er den case-sensitive? Isåfall ska det vara ett stort o istedet. ☺️

Reply:
Kjørte lokalt, men rettet for sikkerhetsskyld.